ui/webui/test/README.rst

```diff
@@ -243,3 +266,58 @@ development environment that's independent from the host::
 
     toolbox create --image quay.io/cockpit/tasks -c cockpit
     toolbox enter cockpit
+
+Web UI End to end tests
+=======================
+
+End to end tests are stored in ``./ui/webui/test/integration``
```

Comment:
Here is a rephrase with improved readability

The end-to-end tests, along with tplib test cases and test plans (only required when executing tests using Permian), are located in the ./ui/webui/test/integration directory.